janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server-configuration.yaml

```diff
@@ -1,7 +1,7 @@
 host: 0.0.0.0
 port: 8182
 scriptEvaluationTimeout: 30000
-channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+channelizer: org.janusgraph.channelizers.JanusGraphWebSocketChannelizer
```

Comment:
I think it would be more confusing if I put comments in the yaml files. Currently there are two YAMl files, one when using the CMG and one when not. The JG channelizers are in place in the CMG YAML because this is when you can make use of the channelizer, and not in the normal YAML because it can't work there. I think the existence of the separate files is enough differentiation between when one could/should/can't use the JG channelizers (of course along with the appropriate documentation on the matter)-- and I have already added to the CGF setting on when it is appropriate to use the JG channelizers and how to configure the YAMl to use so. 

Does that make sense?

Reply:
OK. Make sure to clarify in the docs then. Is there a migration hit for apps that were using CGF in JG 0.2.0?

Reply:
Any app previously using the CGF can make use of this new binding functionality by making use of the new channelizers. That's about it in terms of steps forward for people previously using the CGF.

Reply:
Correct.